setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import glob
+import os
+from setuptools import setup, find_packages
```

Comment:
Is there a reason you use ``setuptools`` rather than ``distutils``? I realize that ``setuptools`` includes the ``find_packages`` routine and other features not in ``distutils``. But it is not a standard module with most user's Python installations, so if you are going to use it we should add it as a pre-requisite for installation in the User's Guide. 

Based on my experience using both ``distutils`` and ``setuptools`` for OpenMOC project, I would recommend using ``distutils`` since it is more stable and reliable (at least I've come across some very subtle issues with deploying C/C++ extension projects using ``setuptools`` in the past). Plus, few if any HPC machines I've come across have ``setuptools`` pre-installed.